internal/controller/storagemgr/storagedriver/storagehandler.go

```diff
@@ -38,11 +38,13 @@ import (
 )
 
 const (
-	deviceNameKey     = "deviceName"
-	resourceNameKey   = "resourceName"
-	apiResourceRoute  = clients.ApiBase + "/resource/{" + deviceNameKey + "}/{" + resourceNameKey + "}"
-	handlerContextKey = "StorageHandler"
-	configPath        = "res/configuration.toml"
+	deviceNameKey            = "deviceName"
+	resourceNameKey          = "resourceName"
+	numberOfReadings         = "readingCount"
+	apiResourceRoute         = clients.ApiBase + "/resource/{" + deviceNameKey + "}/{" + resourceNameKey + "}"
+	apiResourceRouteMultiple = clients.ApiBase + "/resource/{" + deviceNameKey + "}/{" + resourceNameKey + "}/{" + numberOfReadings + "}"
```

Comment:
How about adding `device` route before deviceNameKey like below?

```suggestion
	apiResourceRoute         = clients.ApiBase + "/device/{" + deviceNameKey + "}/resource/{" + resourceNameKey + "}"
	apiResourceRouteMultiple = clients.ApiBase + "/device/{" + deviceNameKey + "}/resource/{" + resourceNameKey + "}/{" + numberOfReadings + "}"
```

Reply:
Thanks ! Done the change.